lib/src/scaffold.dart

```diff
@@ -542,7 +548,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
   }
 
   Widget _buildFrontPanel(BuildContext context) {
-    return Material(
+    var frontPanel = Material(
```

Comment:
make it `final`

```suggestion
    final frontPanel = Material(
```

Reply:
are these changes you'd like me to make or are you guys gonna handle it? I'm new to pull requests...